mess/workflow/rules/processing/reads.smk

```diff
@@ -537,6 +540,146 @@ if not SKIP_SHUFFLE:
             paste -d '\t' <(seqkit seq -n {output}) <(seqkit seq -n {input}) > {log[1]} 
             """
 
+if ERRFREE:
+    rule fix_art_sam_ef:
```

Comment:
We use the fix_art_sam rule to fix #22. Since error free SAM files do not have sequencing errors, we don't need this rule